src/answer.ts

```diff
@@ -37,21 +37,52 @@ const template = renderTemplate`
         }
     </script>
     <style>
-        img {
+        img:not(.origin) {
             width: 100%;
         }
         figure {
             margin: 1.4em 0;
         }
+        .container {
+            display: flex;
+        }
+
+        .avatar {
+            display: flex;
+            flex-direction: column;
+            justify-content: center; /*  头像垂直居中 */
+            align-items: center;     /*  头像水平居中  */
+        }
+
+        .content {
+            margin-left: 10px; /*  与头像的间距 */
+            flex: 1; /* 关键：让 content 占据剩余空间 */
+        }
+
+        .top, .bottom {
+            flex: 1; /*  让顶部和底部内容均分高度  */
+        }
+
     </style>
 </head>
 <body style="max-width: 1000px; margin: 0 auto; padding: 0 1em 0 1em;">
     <header>
         <h1><a href="${"url"}">${"title"}</a></h1>
-        <h2 rel="author">
-		<a href="${"author_url"}" target="_blank">@${"author"}</a>
-		</h2>
-		<p> ${"headline"} </p>
+        <div class="container">
+            <div class="avatar">
+                <img class="origin" src="${"avatar_url"}" />
+            </div>
+            <div class="content">
+                <div class="top">
+                    <h2 rel="author">
+                        <a href="${"author_url"}" target="_blank">@${"author"}</a>
+                    </h2>
+                </div>
```

Comment:
Remove the wrapper div and apply the style directly on the h2 element, then you don't need the `top` and `bottom` class, just:

```css
.author > * {
    flex: 1
}
```